main.go

```diff
@@ -20,33 +20,25 @@ limitations under the License.
 package main
 
 import (
-	"os"
 	"time"
 
-	// Import the snap plugin library
 	"github.com/intelsdi-x/snap-plugin-collector-iostat/iostat"
+	"github.com/intelsdi-x/snap-plugin-lib-go/v1/plugin"
+)
 
-	// Import our collector plugin implementation
-	"github.com/intelsdi-x/snap/control/plugin"
+const (
+	pluginName    = "iostat"
```

Comment:
Please choose one place to store version and name of plugin - here in `main.go` or in `iostat/iostat.go` and remove the other.